src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/GetAzureExpressRouteCircuitCommand.cs

```diff
@@ -37,9 +37,8 @@ public class GetAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
         [ValidateNotNullOrEmpty]
         public virtual string ResourceGroupName { get; set; }
 
-        public override void ExecuteCmdlet()
+        public override void Execute()
         {
-            base.ExecuteCmdlet();
```

Comment:
you shouldnt remove this from NRP cmdlets. (probably all the other cmdlets as well). safer to keep this in the CR so the functionality will not change